feature-pack/src/main/resources/modules/system/layers/base/org/ow2/asm/asm/main/module.xml

```diff
@@ -22,13 +22,13 @@
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.5" name="asm.asm">
+<module xmlns="urn:jboss:module:1.5" name="org.ow2.asm.asm">
```

Comment:
I think module should be named only org.ow2.asm, without second asm repeated.
or we could even stay with old name.

but beyond that it looks ok.

Reply:
That was my first idea, then I saw that there were several artifacts under the group id org.ow2.asm so i wanted to be able to add those